clients/recherche-entreprise/siren.ts

```diff
@@ -15,4 +19,44 @@ const clientComplementsSireneOuverte = async (
   throw new HttpNotFound(siren);
 };
 
-export default clientComplementsSireneOuverte;
+export const clientEtablissementSireneOuverte = async (
+  siret: string
+): Promise<IEtablissement> => {
+  const { results } = await clientSearchSireneOuverte(siret, 1);
+  const result = results[0];
+  return result.matchingEtablissements[0];
+};
+
+export const clientUniteLegaleSireneOuverte = async (
+  siren: Siren,
+  page = 1
+): Promise<IUniteLegale> => {
+  const { results } = await clientSearchSireneOuverte(siren, 1);
```

Comment:
Less relevant as you already test it in `clientSearchSireneOuverte`

But still I would add `try ... catch { throw new httpNotFound(siret) }`just for extra safety